core/dbt/parser/manifest.py

```diff
@@ -442,6 +477,23 @@ def load(self):
                 self.manifest._parsing_info.static_analysis_path_count
             )
 
+        # Load dependencies, load the publication artifacts and create the external nodes.
+        # Reprocess refs if changes.
+        public_nodes_changed = False
+        if skip_parsing:
+            # If we didn't skip parsing, this will have already run because it must run
+            # before process_refs. If we did skip parsing, then it's possible that only
+            # publications have changed and we need to run this to capture that.
+            self.manifest.build_parent_and_child_maps()
+            public_nodes_changed = self.build_public_nodes()
+            if public_nodes_changed:
+                self.manifest.rebuild_ref_lookup()
+                self.process_refs(self.root_project.project_name)
+                # parent and child maps will be rebuilt by write_manifest
+
+        if not skip_parsing or public_nodes_changed:
+            # Following adds publications to manifest too...
```

Comment:
is this comment still applicable? my understanding is that publications are now added to the manifest in build_public_nodes

Reply:
Yep, bad comment. Have changed.